draft-irtf-cfrg-frost.md

```diff
@@ -282,6 +278,9 @@ We now detail a number of member functions that can be invoked on `G`.
 - DeserializeScalar(buf): Attempts to map a byte array `buf` to a `Scalar` `s`.
   This function can raise a DeserializeError if deserialization fails; see
   {{ciphersuites}} for group-specific input validation steps.
+- Scalar(x): Converts a two byte unsigned integer (uint16) `x` to the corresponding 
```

Comment:
Why is this restricted to two-byte integer inputs?

Reply:
I think `Scalar(x)` is supposed to convert identifiers to scalars, so the restriction is due to the identifier restriction

Reply:
In this PR we also explicitly define Identifiers as Scalars, not integers:

![Screenshot_20220907-130739.png](https://user-images.githubusercontent.com/552961/188938469-027cbfc5-bb93-4551-adc0-1f4c80e5fe6a.png)

Reply:
I think this should accept _any_ integer and convert it to a Scalar. There should be no restriction on the size of the input integer. The output Scalar is only a valid identifier if it's in the range [1, MAX_SIGNERS].

Reply:
Done